assets/css/ionicons.css

```diff
@@ -0,0 +1,1479 @@
+@charset "UTF-8";
```

Comment:
For sure, that's a seperate pull request. Let's merge this in for now.

On Tue, Apr 19, 2016 at 10:32 PM, Lindsey <notifications@github.com> wrote:

> In assets/css/ionicons.css
> <https://github.com/ubcgif/geoscixyz-website/pull/1#discussion_r60352190>:
>
> > @@ -0,0 +1,1479 @@
> > +@charset "UTF-8";
>
> can we use font awesome instead?
> http://fortawesome.github.io/Font-Awesome/get-started/
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ubcgif/geoscixyz-website/pull/1/files/dc9e164d7f21ec284ae0f80ab62bccde11b27350#r60352190>
>